Wyvarn/space-xplorer-server
# Mutation Resolvers

## Description

Creates mutation resolvers to be mapped to the Schema block Mutation. This handles mutations which allows users to signup, login, book trips & cancel them

Additional Updates:
1. This fixes a lot of the CI builds when publishing to NPM registry & Docker image builds.
2. Additionally adds mutation resolver to the server instance.

## How can we test this?

1. Run `docker-compose up` to run database
2. Ensure you have performed necessary migrations, details are in the README
3. Run dev server with `yarn start`

## Checklist:
